src/Planet/Planet.jl

```diff
@@ -114,6 +117,23 @@ function e_int_i0 end
 function press_triple end
 """ Surface tension coefficient of water (J/m2) """
 function surface_tension_coeff end
+""" Surface tension coefficient of water (J/m2) """
+function surface_tension_coeff end
+
+#=
+The standard entropy value for dry air is computed based
+on the reference data given in Lemmon et al. [2000], and
+the standard entropy value for water vapor is based on the
+reference data given in Chase [1998].
+=#
+
+""" Entropy of dry air J / (kg K)"""
+function entropy_dry_air end
+""" Entropy of water vapor J / (kg K)"""
+function entropy_water_vapor end
+""" Standard temperature (K)"""
```

Comment:
Maybe better to make explicit that this is the reference temperature for entropies (we have several other reference temperatures). E.g., 
```suggestion
""" Entropy reference temperature (K)"""
```